index.html

```diff
@@ -38,10 +38,35 @@
                 </svg>
                 <div class="github-link-text">Star my repo!</div>
             </a>
+
             <button
                 style="background-color: #347d39; color:#c6c7c7; text-decoration: none; border-radius: 0.5rem; padding: 0.5rem 1rem; cursor: pointer;  border: none; position: absolute; top: 0; right: 0; margin: 1rem"
-                @click="clearSaveData()" v-if="saveData.length > 0">Clear save data</button>
-            <h1 style="text-align: center; cursor: pointer;" @click="navigateTo('')">Infinite Craft Wiki</h1>
+                @click="clearSaveData()" v-if="saveData.length > 0">
+                Clear save data
+            </button>
+
+            <h1 style="text-align: center; cursor: pointer; margin-bottom: 0px;" @click="navigateTo(''); searchQuery='';">
```

Comment:
There's some cursor pointer issues here that aren't present in the current site. Not sure what's up with that